lib/husky_musher/templates/registration_required.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h2>{% block title %}Registration Required{% endblock %}</h2>
+{% if not registration_complete %}
+    {% if not redcap_record_exists %}
+        <p>
+            <b>{{ netid }}</b> has not signed up!
+        </p>
+    {% elif not registration_complete %}
+        <p>
+            <b>{{ netid }}</b> has not finished signing up for Husky Coronavirus Testing!
+        </p>
+    {% endif %}
+    <p>
+        Please visit <a href="http://seattleflu.org/uw/start">http://seattleflu.org/uw/start</a>
+        to sign up.
+    </p>
+{% else %}
+    <p>
+        <b>{{ netid }}</b> is enrolled.
+    </p>
```

Comment:
You're right, thanks. That's some cruft left over from when I was first developing this app. I think I'm personally fine removing the if statement for `if not registration_complete`, since we only render this template in `app.py` if registration is not complete.